src/tools/dev/doxygenate/doxygenate.py

```diff
@@ -191,7 +191,10 @@ def get_visit_svn_rev(src):
     Attempts to get the VisIt SVN Reversion number from the source tree.
     """
     svnrev = "<Unknown>"
-    proc = subprocess.Popen('svn info %s' % src, shell=True,stdout=subprocess.PIPE)
+    proc = subprocess.Popen('svn info %s' % src,
```

Comment:
references svn. Should this be reworked for git?

Reply:
We probably don't use doxygenate tool that often but, yes, what you suggest makes sense. 

`svn info` produces something like...

```
% svn info
Path: .
Working Copy Root Path: /Users/miller86/silo/trunk
URL: svn+ssh://miller86@surface.llnl.gov/usr/gapps/silo/svn/trunk
Relative URL: ^/trunk
Repository Root: svn+ssh://miller86@surface.llnl.gov/usr/gapps/silo/svn
Repository UUID: 92378438-2262-0410-827d-edb41c1c42c8
Revision: 894
Node Kind: directory
Schedule: normal
Last Changed Author: miller86
Last Changed Rev: 891
Last Changed Date: 2021-09-12 11:22:53 -0700 (Sun, 12 Sep 2021)
```

and is somewhat equiv to `git remote -v`

Reply:
Or, `git remote show origin`